std/exception.d

```diff
@@ -1478,6 +1478,34 @@ private bool isUnionAliasedImpl(T)(size_t offset)
         static assert( isUnionAliased!(S.A5, 1)); //a5.b1;
 }
 
+package string errnoString(int errno) nothrow @system
```

Comment:
Why `@system`? This would prevent using this function in `@safe` functions without hacks.